source/agora/common/Transaction.d

```diff
@@ -243,7 +243,7 @@ public Amount getSumOutput (Transaction tx) nothrow pure @safe @nogc
 
 *******************************************************************************/
 
-public bool verifyData (Transaction tx,
+public bool verify (Transaction tx,
```

Comment:
I wish `in` meant `scope const ref`... Because `scope const ref`

Reply:
Do you need me to const-ify the parameters?

Reply:
It would be great